enterprise/utils.py

```diff
@@ -1451,15 +1450,29 @@ def enroll_users_in_course(
     return successes, pending, failures
 
 
+def is_valid_url(url):
+  try:
+    result = urlparse(url)
+    return all([result.scheme, result.netloc])
+  except ValueError:
+    return False
+
+
 def get_platform_logo_url():
     """
-    Return an absolute URL of the platform logo using the branding api
+    Return an absolute URL of the platform logo using the branding api.
     """
     # Return fake URL for tests rather than mock get_logo_url for every test using EnterpriseCustomer
-    return urljoin(
-        settings.LMS_ROOT_URL,
-        get_logo_url()
-    ) if get_logo_url else 'http://fake.url'
+    fake_url = 'http://fake.url'
+    logo_url = get_logo_url()
+    if not logo_url:
+        return None
```

Comment:
would like to return `fake_url` url here?

Reply:
Based on the comment "Return fake URL for tests rather than mock get_logo_url for every test using EnterpriseCustomer", I think the `fake_url` was only included here for tests? I think it would be odd to actually return `http://fake.url` for real as that's not a valid URL, so might as well just return `None`.

I will remove that comment now that the test has been updated.

Reply:
Ok got it.